src/troubleshooting/errors.rst

```diff
@@ -0,0 +1,118 @@
+
+TSIG error with server: tsig verify failure
+********************************************
+
+.. code-block:: bash
+
+  (Fri Apr 14 16:37:19 2023) example.com sssd[87108]: ; TSIG error with server: tsig verify failure
+
+When SSSD is configured with id_provider = ad, by default, sssd will try to update the DNS record using the nsupdate command (the default value of the dyndns_update option is true). If, for some reason, tsig/nsupdate is failing, sssd will return ``TSIG error with server: tsig verify failure``. This error can be safely ignored if ``Dynamic DNS update`` is not being used, or if  ``Dynamic DNS update`` can be disabled by using ``dyndns_update = false`` under the ``[$domain]`` section of the ``/etc/sssd/sssd.conf`` file. If ``Dynamic DNS update`` is being used, then in order to identify the issue, enable sssd debugging using https://sssd.io/troubleshooting/basics.html.
+
+SSSD is offline/Backend is offline
+***********************************
+
+.. code-block:: bash
+
+  (Sat Apr 15 01:07:19 2023) [sssd[be[example.com]] [sbus_issue_request_done] (0x0040): sssd.dataprovider.getAccountInfo: Error [1432158212]: SSSD is offline
+
+SSSD is going offline because it cannot establish a connection to the LDAP server, but the cause could vary. It may be a DNS issue where SRV records are not resolving. It may be a connection issue when the remote server is unreachable because it is behind a firewall, etc.
+
+s2n exop request failed/ldap_extended_operation failed
+*******************************************************
+
+.. code-block:: bash
+
+  (Sat Apr 15 12:07:19 2023): [be[ipa.example.com]] [sdap_call_op_callback] (0x20000): [RID#9] Handling LDAP operation [26][server: [10.23.x.x:389] IPA EXOP] took [10083.267] milliseconds.
+  (Sat Apr 15 12:07:19 2023): [be[ipa.example.com]] [ipa_s2n_exop_done] (0x0040): [RID#9] ldap_extended_operation result: Time limit exceeded(3), (null).
+  (Sat Apr 15 12:07:19 2023): [be[ipa.example.com]] [ipa_s2n_exop_done] (0x0040): [RID#9] ldap_extended_operation failed, server logs might contain more details.
+  (Sat Apr 15 12:07:19 2023): [be[ipa.example.com]] [sdap_op_destructor] (0x2000): [RID#9] Operation 26 finished
+  (Sat Apr 15 12:07:19 2023): [be[ipa.example.com]] [ipa_s2n_get_user_done] (0x0040): [RID#9] s2n exop request failed.
+  (Sat Apr 15 12:07:19 2023): [be[ipa.example.com]] [sdap_id_op_done] (0x4000): [RID#9] releasing operation connection
+  (Sat Apr 15 12:07:19 2023): [be[ipa.example.com]] [ipa_subdomain_account_done] (0x0040): [RID#9] ipa_get_*_acct request failed: [1432158230]: Network I/O Error.
+  
+ldap_extended_operation failed or s2n exop request failed indicates that the IPA client has sent the request to the IPA server, but due to some reason the request has been failed. The IPA server's sssd log will contain more information. For more details, refer to https://sssd.io/troubleshooting/ipa_provider.html
```

Comment:
Worth mentioning here you should compare the logs between both client and server from the same timeframe. The IPA client s2n request also hits the IPA directory server extdom plugin, investigating IPA 389 directory server access/error logs may be useful here too.